chat-widget/src/components/postchatsurveypanestateful/PostChatSurveyPaneStateful.tsx

```diff
@@ -109,6 +111,12 @@ export const PostChatSurveyPaneStateful = (props: IPostChatSurveyPaneStatefulPro
                     Description: "Customer Voice form response error.",
                     ExceptionDetails: { message: "Customer Voice form response error." }
                 });
+            } else if (typeof(data) === "string" && data.startsWith(CustomerVoiceEvents.FormsError)) {
+                TelemetryHelper.logActionEventToAllTelemetry(LogLevel.ERROR, {
+                    Event: TelemetryEvent.CustomerVoiceFormsError,
+                    Description: "Customer Voice forms error.",
+                    ExceptionDetails: { message: `Customer Voice forms error: ${data}` }
```

Comment:
NIT:  the string is duplicated from Description.

Reply:
Updated